src/lib.rs

```diff
@@ -0,0 +1 @@
+pub mod docker;
```

Comment:
nit: we might want to flatten the `docker` directory to remove this extra `docker` module